package.json

```diff
@@ -27,5 +27,8 @@
   },
   "scripts": {
     "test": "mocha --reporter spec"
+  },
+  "dependencies": {
+    "is-plain-object": "=3.0.1"
```

Comment:
Not a fan of introducing external dependencies for such a small change, can we do this internally instead?